pkg/webservice/handlers.go

```diff
@@ -516,14 +516,23 @@ func isChecksumEqual(checksum string) bool {
 
 func checkHealthStatus(w http.ResponseWriter, r *http.Request) {
 	writeHeaders(w)
-	metrics := metrics2.GetSchedulerMetrics()
-	result := scheduler.GetSchedulerHealthStatus(metrics, schedulerContext)
-	if !result.Healthy {
-		log.Logger().Error("Scheduler is not healthy", zap.Any("health check values", result.HealthChecks))
-		buildJSONErrorResponse(w, "Scheduler is not healthy", http.StatusServiceUnavailable)
-	}
-	if err := json.NewEncoder(w).Encode(result); err != nil {
-		buildJSONErrorResponse(w, err.Error(), http.StatusInternalServerError)
+
+	// Fetch last healthCheck result
+	result := schedulerContext.GetLastHealthCheckResult()
+	if result != nil {
+		if !result.Healthy {
+			log.Logger().Error("Scheduler is not healthy", zap.Any("health check info", *result))
+			buildJSONErrorResponse(w, "Scheduler is not healthy", http.StatusServiceUnavailable)
+		} else {
+			log.Logger().Info("Scheduler is healthy", zap.Any("health check info", *result))
+			buildJSONErrorResponse(w, "Scheduler is healthy", http.StatusOK)
```

Comment:
When scheduler is healthy, we shouldn't log it with `ERROR` level.

Reply:
done